v19.2/alter-type.md

```diff
@@ -4,11 +4,15 @@ summary: Use the ALTER TYPE statement to change a column's data type.
 toc: true
 ---
 
-The `ALTER TYPE` [statement](sql-statements.html) is part of [`ALTER TABLE`](alter-table.html) and changes a column's [data type](data-types.html).
+The `ALTER TYPE` [statement](sql-statements.html) is part of [`ALTER TABLE`](alter-table.html).
 
-{% include {{ page.version.version }}/sql/combine-alter-table-commands.md %}
+{{site.data.alerts.callout_info}}
+In CockroachDB versions < v20.2, support for altering column types is limited to increasing precision. For details, see [Limitations](#limitations).
+{{site.data.alerts.end}}
 
-## Considerations
+## Limitations
+
+In CockroachDB versions < v20.2, support for altering column types is limited to increasing the precision of the current type of a column. You cannot convert the column type to another data type, or decrease the precision of the column type.
```

Comment:
Is it worth mentioning there is no errors for no-ops as well - changing the column type from the current type to itself.